index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Trial Project</title>
+    <style>
+      .bt {
+        background-color: #04aa6d;
+        font-size: large;
+        width: 100%;
+        padding: 12px 20px;
+        font-weight: bold;
+        border: none;
+        color: aliceblue;
+        border-radius: 3px;
+      }
+
+      body {
+ background-image: url("bg.jpg");
+ background-color: #9b9ba7;
+}
+      .b1 {
+        font-size: 36px;
+        font-weight: bold;
+        line-height: 0.25em;
+        text-align: center;
+        color: rgb(233, 233, 233);
+      }
+      .b2 {
+        text-align: center;
+        color: rgb(233, 233, 233);
+        font-style: italic;
+      }
+      .b3 {
+        font-size: 18px;
+        font-weight: bold;
+      }
+
+      .container {
+        max-width: 650px;
+        margin: 0 auto;
+        padding: 30px 30px;
+        border: 1px solid rgb(23, 23, 43);
+        background-color: rgba(5, 5, 31, 0.7);
+        color: white;
+        border-radius: 5px;
+      }
+      #name,#email,#number,#dropdown{
+        width: 100%;
+        height: 2.35rem;
+      }
+      .text-area{
+            width: 99%;
+            min-height: 120px;
+            border-radius: 3px;
+            resize: vertical;
+      }
+    </style>
+  </head>
+
+  <body>
+    <h1 class="b1" id="title">freeCodeCamp Survey Form</h1>
```

Comment:
Please make your class more descriptive.

Reply:
@yohannesdagnachew,
You can add a comment as a suggestion, and give reasons for your change request. in this case, you can create a comment like this:
- Your class name is not descriptive, please consider using a more descriptive name, it will make your code more readable